### PR TITLE
refactor(cloud-classic): migrate traefik to v2

### PR DESCRIPTION
**Pull request by njfamirm**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

Fixes #(issue)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Open an issue for bug report or feature request and mention it to this PR.
- [x] The commit message follows our guidelines: <https://github.com/AliMD/alwatr/blob/main/CONTRIBUTING.md#commit-message-format>
- [x] All methods have proper description in `jsdocs` format with `@example` section.
- [x] Docs have been added / updated (for bug fixes / features).
- [ ] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by njfamirm** [comment]
اینو تست نکردم هنوز

**Comment by njfamirm** [comment]
```yaml
networks:
  alwatr-public-network:
    name: alwatr-public-network
    external: true

services:
  service-proxy:
    image: ${TRAEFIK_IMAGE}

    hostname: traefik-proxy.alwatr.io

    networks:
      - alwatr-public-network

    restart: unless-stopped

    cpu_shares: 4096
    mem_limit: 256m

    logging:
      driver: json-file
      options:
        max-size: '10m'
        max-file: '2'

    ports:
      - ${HOST}:80:80
      - ${HOST}:443:443

    volumes:
      - type: bind
        source: /var/run/docker.sock
        target: /var/run/docker.sock
      - type: bind
        source: ./traefik.toml
        target: /traefik.toml
      - type: bind
        source: ./_data/acme.json
        target: /acme.json

    labels:
      traefik.enable: false
      traefik.port: 80
      traefik.http.routers.router0.rule: Host(${DOMAIN})
      traefik.http.middlewares.auth.basicauth.users: ${AUTH}

  whoami:
      image: "traefik/whoami"
      container_name: "simple-service"
      labels:
        traefik.enable: true
        traefik.http.routers.whoami.rule: Host(`whoami.localhost`)
        traefik.http.routers.whoami.entrypoints: websecure
        traefik.http.routers.whoami.tls.certresolver: default

```

**Comment by njfamirm** [comment]
Screen Shot 1401-05-29 at 00 32 50

**Comment by njfamirm** [comment]
@alimd

> Screen Shot 1401-05-29 at 00 32 50

File: packages/classic-cloud/traefik-proxy/compose.yml [hunk 1]
```diff
@@ -24,8 +24,8 @@ services:
         max-file: '2'
 
     ports:
-      - ${HOST}:80:80
-      - ${HOST}:443:443
+      - 80:80
+      - 443:443
```
Comment (njfamirm) [inline packages/classic-cloud/traefik-proxy/compose.yml:28]: مشکل اینجا بود!!!!
نمی فهممش

**Comment by AliMD** [comment]
یک فکری برای این بکن
یا ببندیمش یا بگو کجا گیره کمک کنم
خلاصه یه جوری تمام بشه

**Comment by njfamirm** [comment]
دوباره میرم سراغش

**Comment by njfamirm** [comment]
isj

**Pull request closed without merging**